ramls/marccat.raml

```diff
@@ -110,10 +110,10 @@ resourceTypes:
           searchable: {description: "using CCL query in q parameter",
           example: "q=na \"giannini\"}
```

Comment:
The RAML error seems to be the missing double-quote on line 111.

Reply:
Thanks a lot